app/views/users/index.html.erb

```diff
@@ -1,5 +1,6 @@
-<section class="users">
-  <ul class="users__list">
+  <%= link_to "New Post", new_user_post_path(user_id: @current_user.id), class: "btn-primary"  %>
```

Comment:
There's an issue with your parameters on your `New Post` link. Kindly look into it.

![image](https://user-images.githubusercontent.com/38088818/185680231-7c036cb6-755e-4b9c-9915-ad42c8e6a8e1.png)

Reply:
Hi, @KDlamini thank you for your feedback. This error could be happening because there are no records in the database and is trying to assign the `@current_user` to the first record in the `Users` table, getting `nil` as a result. Did you run `bin/rails db:seed`? I got some dummy data.